src/irmin-pack/pack_store.ml

```diff
@@ -517,6 +518,11 @@ module Maker (Index : Pack_index.S) (K : Irmin.Hash.S with type t = Index.key) :
         close t.pack)
 
     let close t =
+      (* It is expected that a flush was performed before [close]. Otherwise
+         data is lost. (ext takes care about it).
+
+         This function does not flush because index is expected to already be
+         closed. *)
```

Comment:
```suggestion
      (* This function does not flush because: 1/ index is expected to already be closed and 2/ the pack file was flushed upstream by [ext]. *) 
```